.github/workflows/node-js.yaml

```diff
@@ -0,0 +1,25 @@
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 16
+      uses: actions/setup-node@v2
+      with:
+        node-version: 16
+        cache: 'npm'
+    - run: npm ci
+    - run: npm run lint
+    - run: npm run build
```

Comment:
We now have continuous integration in the cloud !

Reply:
We don't have tests yet but eslint + typescript in strict mode is already a good start.